common/src/main/scala/azure/WNSRawPush.scala

```diff
@@ -1,16 +1,14 @@
 package azure
 import play.api.http.Writeable
-import play.api.libs.ws.{WSRequest, WSResponse}
+import play.api.libs.ws.{DefaultBodyWritables, WSRequest, WSResponse}
 import utils.WriteableImplicits._
 
 import scala.concurrent.Future
 
-case class WNSRawPush(body: String, tags: Option[Tags]) extends RawPush {
+case class WNSRawPush(body: String, tags: Option[Tags]) extends RawPush with DefaultBodyWritables {
```

Comment:
I think I'd rather see it as a import
``` scala
import play.api.libs.ws.DefaultBodyWritables._
```

It should give you the same result, and that avoids tying all the members of `DefaultBodyWritables` to each instance of WNSRawPush